addons/web/static/tests/fields/relational_fields/field_one2many_tests.js

```diff
@@ -5674,6 +5662,42 @@ QUnit.module('fields', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('one2many list edition, no "Remove" button in modal', async function (assert) {
+            assert.expect(2);
+
+            this.data.partner.fields.foo.default = false;
+
+            var form = await createView({
+                View: FormView,
+                model: 'partner',
+                data: this.data,
+                arch: '<form string="Partners">' +
+                    '<field name="p">' +
+                    '<tree>' +
+                    '<field name="foo"/>' +
+                    '</tree>' +
+                    '<form string="Partners">' +
+                    '<field name="display_name"/>' +
+                    '</form>' +
+                    '</field>' +
+                    '</form>',
```

Comment:
could you indent the arch? Also, the string attribute on form nodes is not necessary